en/flight_modes_vtol/mission.md

```diff
@@ -0,0 +1,42 @@
+# Mission Mode (VTOL)
+
+[<img src="../../assets/site/position_fixed.svg" title="Global position fix required (e.g. GPS)" width="30px" />](../getting_started/flight_modes.md#key_position_fixed)
+
+_Mission mode_ causes the vehicle to execute a predefined autonomous [mission](../flying/missions.md) (flight plan) that has been uploaded to the flight controller.
+The mission is typically created and uploaded with a Ground Control Station (GCS) application like [QGroundControl](https://docs.qgroundcontrol.com/master/en/) (QGC).
+
+VTOL vehicles follow the behavior and parameters of fixed-wing when in FW mode, and of multicopter when in MC mode.
+For more information see the specific docs for each mode:
+
+- [Mission Mode (MC)](../flight_modes_mc/mission.md)
+- [Mission Mode (FW)](../flight_modes_fw/mission.md)
+
+The following sections outline mission mode behaviour that is VTOL specificL.
+
+## Mission Commands
+
+PX4 "accepts" the mission commands for fixed wing or multicopter when in those modes.
+
+The following VTOL-specific commands are as defined in the MAVLink specification.
+
+- [MAV_CMD_NAV_VTOL_TAKEOFF](https://mavlink.io/en/messages/common.html#MAV_CMD_NAV_VTOL_TAKEOFF)

```

Comment:
What does VTOL do if it gets the normal takeoff and land commands used by FW/MC?

Reply:
If it's in hover: it will simply execute the takeoff/landing like a MC.
If it's in FW: it will not be possible to takeoff. And the landing will be transformed to a VTOL_LAND unless NAV_FORCE_VT is set to false.

Reply:
UPdated to capture this.